src/app/lib/utilities/ichefURL/index.test.js

```diff
@@ -0,0 +1,24 @@
+import getIChefURL from '.';
+
+describe('getIchefURL', () => {
+  it('builds ichef img url based on originCode, locator, resolution passed', () => {
+    const input = {
+      originCode: 'cpsprodpb',
+      locator: 'cc66/live/5b34d420-b382-11e9-b6fd-e3056fffd1f1.jpg',
+      resolution: '660',
+    };
+    const expectedOutput =
+      'https://ichef.bbci.co.uk/news/660/cpsprodpb/cc66/live/5b34d420-b382-11e9-b6fd-e3056fffd1f1.jpg';
+
+    expect(getIChefURL(input)).toEqual(expectedOutput);
+  });
+
+  it('builds a valid img url based on the scr url we pass', () => {
+    const locator = 'ichef.test.bbci.co.uk/images/ic/$widthxn/p01k6mtv.jpg';
+    const originCode = 'mpv';
+    const expectedOutput =
+      'https://ichef.test.bbci.co.uk/images/ic/512xn/p01k6mtv.jpg';
+
+    expect(getIChefURL({ locator, originCode })).toEqual(expectedOutput);
+  });
```

Comment:
The tests look good. Could you please add a test case with `originCode = 'pips';`, to test the fork in logic in the `getIChefURL` function?

Reply:
Done

Reply:
Thanks.